internal/librarian/generate_test.go

```diff
@@ -249,7 +304,8 @@ func TestRunConfigureCommand(t *testing.T) {
 				containerClient: test.container,
 			}
 
-			if test.name == "configures library" {
+			if test.name == "configures library successfully" ||
```

Comment:
nit: This code was already here so don't worry for now but I would suggest not using this pattern in the future where we branch a table test on its name. In such a case it is likely cleaner to just create another whole test.

Reply:
Sure, thanks for the suggestion.